src/pages/workspaces/workspace/analysis/useAnalysisFiles.ts

```diff
@@ -100,7 +100,8 @@ export const useAnalysisFiles = (): AnalysisFileStore => {
 
   useEffect(() => {
     refresh()
-  }, [workspace]) // eslint-disable-line react-hooks/exhaustive-deps
+  }, [workspace.workspace]) // eslint-disable-line react-hooks/exhaustive-deps
+  // refresh depends only on workspace.workspace, do not want to refresh on workspace.workspaceInitialized
```

Comment:
When the Google permissions checking completes, `workspace.workpaceInitialized` is set to true. This was causing `refresh` to be called a second time (delayed further in this PR from when it previously as because we are making more ajax calls to determine if permissions are fully synced), which led to a spinner often interfering with pressing the "Start" button to create an Analysis during the integration tests.

The ignoring of the exhaustive-deps warning is not good-- it would be much better if `refresh` can be wrapped with `useCallback` or `useMemo`, and then the actual dependencies listed.